### ISSUE TITLE
Allow for multiple sources per spell id

### ISSUE THREAD

**Issue by Rainrider** [AdiAddons/LibPlayerSpells-1.0#83]
Mortal Wounds (115804) can be applied by monks' Rising Sun Kick and hunter pets. This causes an error in __RegisterSpells:
```
1x LibPlayerSpells-1.0-7: 1 errors in MONK database:
Spell #115804: already listed in the HUNTER database
LibPlayerSpells-1.0-7\data\Monk.lua:23: in main chunk
```

**Comment by Rainrider** [comment]
This has been introduced by https://github.com/AdiAddons/LibPlayerSpells-1.0/commit/f065648a7fd63402c8b351f964acf34f66b95b54

**Comment by Rainrider** [comment]
A working trial is under https://github.com/Rainrider/LibPlayerSpells-1.0/tree/sources. Needs review though

**Comment by Rainrider** [comment]
Should be resolved as of ffe8530311d3dcb634e9c14b46aeffdbcc4409f7
